Chemical - Fix chemical zeus module

AtrixZockt opened the pull request:
I hate this bs...

**When merged this pull request will:**
- _Describe what this pull request will do_
- _Each change in a separate line_

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

AtrixZockt committed:
Fixed chemical zeus module

I hate this bs...

YetheSamartaka commented on addons/chemical/functions/fnc_gasCheck.sqf:
```diff
@@ -18,70 +17,89 @@
+
+params ["_logic", "_pos", "_radius_max", "_radius_min", "_gastype"];
+
+systemChat "Gas check started";
```
left systemChat from debugging

YetheSamartaka commented on addons/chemical/functions/fnc_gasCheck.sqf:
```diff
@@ -18,70 +17,89 @@
+systemChat "Gas check started";
 
-params ["_logic","_pos","_radius_max","_radius_min","_gastype"];
+systemChat str _logic;
```
here as well

YetheSamartaka commented on addons/chemical/functions/fnc_gasCheck.sqf:
```diff
@@ -18,70 +17,89 @@
-                private _arrPos = _playerARR find _x;
-                _playerARR deleteAt _arrPos;
-                _logic setVariable [QGVAR(gas_playerArr), _playerARR, true];
+                systemChat format["Player treated: %1", _x];
```
systemChat

YetheSamartaka commented on addons/chemical/functions/fnc_gasCheck.sqf:
```diff
@@ -18,70 +17,89 @@
+            if (!(_x in _playerArr) && _distance < _radius_max) then {
+                _playerArr pushBack _x;
+
+                systemChat format["Player in Zone: %1", _x];
```
systemChat

YetheSamartaka commented on addons/chemical/functions/fnc_gasCheck.sqf:
```diff
@@ -18,70 +17,89 @@
+            } else {
+                if (_x in _playerArr && _distance > _radius_max) then {
+
+                    systemChat format["Player left Zone: %1", _x];
```
systemChat

YetheSamartaka commented on addons/chemical/functions/fnc_gasCheckLocal.sqf:
```diff
@@ -23,103 +23,68 @@
 
 params ["_unit", "_logic", "_pos", "_radius_max", "_radius_min", "_gastype"];
 
+systemChat "Local Gas Check started";
```
systemChat

YetheSamartaka commented on addons/chemical/functions/fnc_ui_gasModule.sqf:
```diff
@@ -10,24 +10,28 @@
 
 params ["_control"];
 
+systemChat "Initialize Module Test";
```
systemChat

YetheSamartaka commented on addons/chemical/functions/fnc_ui_gasModule.sqf:
```diff
@@ -10,24 +10,28 @@
-private _logic = missionNamespace getVariable["BIS_fnc_initCuratorAttributes_target",objNull];
+private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+
+systemChat str _ctrlButtonOK;
```
systemChat

YetheSamartaka commented on addons/chemical/functions/fnc_ui_gasModule.sqf:
```diff
@@ -71,10 +75,12 @@ if !(isNull attachedTo _logic) then {
 private _fnc_onConfirm = {
     params [["_ctrlButtonOK", controlNull, [controlNull]]];
 
+    systemChat "Module Confirm Click";
```
systemChat

YetheSamartaka commented on addons/chemical/functions/fnc_ui_gasModule.sqf:
```diff
@@ -93,29 +99,30 @@ private _fnc_onConfirm = {
-            private _position = getPos _object;
 
-            [_logic,_position,_radius_max,_radius_min,_gastype] call FUNC(gasCheck);
+            systemChat "Module Gas Check Initialize";
```
systemChat

YetheSamartaka commented on addons/chemical/functions/fnc_ui_gasModule.sqf:
```diff
@@ -93,29 +99,30 @@ private _fnc_onConfirm = {
         } else {
-            private _position = getPos _logic;
-            [_logic,_position,_radius_max,_radius_min,_gastype] call FUNC(gasCheck);
+            systemChat "Module Gas Check Initialize";
```
systemChat

YetheSamartaka reviewed (changes requested):
Please remove systemChat messages.

AtrixZockt committed:
Removed debug systemchats

AtrixZockt committed:
Pharma - Adjust magazine items to use ace system (#549) (#1)

**When merged this pull request will:**
- Because of "registerItemReplacement" items added via arsenal couldnt
be removed
- Its been a while since ace update containing our feature but it seems
we didnt update it properly

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

AtrixZockt committed:
Merge pull request #2 from KAT-Advanced-Medical/dev-Tomcat

Pharma - Adjust magazine items to use ace system (#549)

MiszczuZPolski committed:
some big changes :skull:

MiszczuZPolski committed:
fix for module plus some formatting

MiszczuZPolski committed:
remove systemchat

AtrixZockt committed:
Fixed - Chemical zeus zone

MiszczuZPolski committed:
yet another zeus module fix :skull:

MiszczuZPolski committed:
fix locality for cs grenade

MiszczuZPolski committed:
Merge branch 'dev-Tomcat' into dev-Tomcat

AtrixZockt committed:
Merge branch 'KAT-Advanced-Medical:dev-Tomcat' into dev-Tomcat

MiszczuZPolski committed:
remove systemChat
